.github/pull_request_template.md

```diff
@@ -1,22 +1,25 @@
-### Thank you for contributing to the Azure IoT Plug and Play Models repository
+### Thank you for contributing to the IoT Plug and Play Models repository
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+:memo: Please review this checklist before submission
 
-The repository pipeline will ensure minimum validation of incoming DTDL models.  
+- [ ] :eyes: The models submitted in this PR follow the [DTDL Naming Guidelines](https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines) for contributing to this repository.
+- [ ] 🕵️I have validated the models locally using the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine) command line tool.
+- [ ] ❗ I understand once the PR is approved, the models are considered immutable: any edits will require to submit new files.
+- [ ] ✨ When creating a new top level `dtmi:namespace`, I have completed the [PR Info Template](./#pr-info-template) below.
```

Comment:
The `./` needs to be removed in `[PR Info Template](./#pr-info-template)` to link to anchor/hash link in the same document, otherwise the link is rooted in the .github directory.